lib/ansible/modules/system/pamd.py

```diff
@@ -13,93 +13,93 @@
+    type: str
   new_control:
     description:
     - The new control to assign to the new rule.
+    type: str
   new_module_path:
     description:
     - The new module path to be assigned to the new rule.
+    type: str
   module_arguments:
     description:
-    - When state is 'updated', the module_arguments will replace existing
-      module_arguments.  When state is 'args_absent' args matching those
-      listed in module_arguments will be removed.  When state is
-      'args_present' any args listed in module_arguments are added if
-      missing from the existing rule.  Furthermore, if the module argument
-      takes a value denoted by '=', the value will be changed to that specified
-      in module_arguments.  Note that module_arguments is a list.  Please see
-      the examples for usage.
+    - When state is C(updated), the module_arguments will replace existing module_arguments.
+    - When state is C(args_absent) args matching those listed in module_arguments will be removed.
+    - When state is C(args_present) any args listed in module_arguments are added if
+      missing from the existing rule.
+    - Furthermore, if the module argument takes a value denoted by C(=),
+      the value will be changed to that specified in module_arguments.
+    type: list
   state:
-    default: updated
-    choices:
-    - updated
-    - before
-    - after
-    - args_present
-    - args_absent
-    - absent
     description:
-    - The default of 'updated' will modify an existing rule if type,
-      control and module_path all match an existing rule.  With 'before',
-      the new rule will be inserted before a rule matching type, control
-      and module_path.  Similarly, with 'after', the new rule will be inserted
-      after an existing rule matching type, control and module_path.  With
-      either 'before' or 'after' new_type, new_control, and new_module_path
-      must all be specified.  If state is 'args_absent' or 'args_present',
-      new_type, new_control, and new_module_path will be ignored.  State
-      'absent' will remove the rule.  The 'absent' state was added in version
-      2.4 and is only available in Ansible versions >= 2.4.
+    - The default of C(updated) will modify an existing rule if type,
+      control and module_path all match an existing rule.
+    - With C(before), the new rule will be inserted before a rule matching type,
+      control and module_path.
+    - Similarly, with C(after), the new rule will be inserted after an existing rulematching type,
+      control and module_path.
+    - With either C(before) or C(after) new_type, new_control, and new_module_path must all be specified.
+    - If state is C(args_absent) or C(args_present), new_type, new_control, and new_module_path will be ignored.
+    - State C(absent) will remove the rule.  The 'absent' state was added in Ansible 2.4.
+    type: str
+    choices: [ absent, before, after, args_absent, args_present, updated ]
```

Comment:
I'm surprised you didn't alphabetize the choices . . . I'm fine with merging this as-is, or changing it, just thought it might bother you in future.

Reply:
```suggestion
    choices: [ absent, after, args_absent, args_present, before, updated ]
```
Haha, I would have done that, but sometimes I can restrain myself from doing more than intended. This was such a time I guess :-P